### PR TITLE
feat: add contributing guidelines and working example

### PR DESCRIPTION
**Pull request by bradvogel**
In reviewing PR https://github.com/bee-queue/arena/pull/324 I realized that there was no easy way to run the queue locally. This adds a basic working example and some simple documentation.

### CODE REVIEW

**Comment by bradvogel** [comment]
fyi @roggervalf

File: example/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Overview
+
+This is a simple demonstration of how to run Area and connect it to [Bee Queue](https://github.com/mixmaxhq/bee-queue).
```
Comment (roggervalf) [inline example/README.md:3]: minor change in Area to be Arena

Comment (bradvogel) [reply example/README.md]: Great catch, thanks https://github.com/bee-queue/arena/pull/326/commits/d315cf90ba39d694a49c8151ec5fd01a85beb61f

**Review by roggervalf — APPROVED** [review]

**Comment by bradvogel** [comment]
@shils when you get a chance

**Review by shils — APPROVED** [review]
Thanks!

**Pull request merged**